.github/workflows/locker.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     if: ${{ github.repository_owner == 'dotnet' }}
     steps:
       - name: Checkout Actions
-        uses: actions/checkout@v4
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
```

Comment:
Usually the comment for these will be the full tag that matches the sha (`vX.Y.Z`). Otherwise over time the `vX` tag will move and make the comment out of date.

Reply:
yeah, this is 4.2.2 https://github.com/actions/checkout/commit/11bd71901bbe5b1630ceea73d27597364c9af683

https://github.com/mheap/pin-github-action/issues/192

I'll edit these manually I guess :\

Reply:
I think step security themselves have a tool that does this, I've used it in the past.

Reply:
done